File: sdk/typescript/src/___PLAYGROUND___/relayer/get-allowlist.ts
import { begin } from '../helpers';

export const getAllowlist = async () => {
  const context = await begin();

  console.log('Getting relayer allowlist...');
  const allowlists = await context.relayer.allowlist.get();
  console.log('Relayer address:', allowlists);

  await context.end();
};

getAllowlist().then(() => console.log('get-allowlist done'));

File: sdk/python/src/__PLAYGROUND__/relayer/get-allowlist.py
import asyncio

from __PLAYGROUND__.helper import begin, end


async def getAllowlist():
    rrelayer_node = None

    try:
        _, relayer, _, rrelayer_node = await begin()

        print("Getting relayer allowlist...")
        allowlists = await relayer.allowlist.get()
        print("AllowLists:", allowlists)
    except Exception as e:
        print("getAllowlist failed:", e)
    finally:
        if rrelayer_node is not None:
            end(rrelayer_node)


if __name__ == "__main__":
    asyncio.run(getAllowlist())
    print("get-allowlist done")
